yarn.lock

```diff
@@ -5085,7 +5085,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@metamask/eth-snap-keyring@npm:^2.1.1, @metamask/eth-snap-keyring@npm:^2.1.2":
+"@metamask/eth-snap-keyring@npm:^2.1.1":
```

Comment:
Note: The previous v2 is still used by `@metamask/accounts-controller`, which means there are now two instances (dupes) of the package.

Reply:
You mean this, right?
```
└─ metamask-crx@workspace:.
   ├─ @metamask/eth-snap-keyring@npm:3.0.0 (via npm:^3.0.0)
   └─ @metamask/accounts-controller@npm:11.0.0 [15918] (via npm:^11.0.0 [15918])
      └─ @metamask/eth-snap-keyring@npm:2.1.2 (via npm:^2.1.1)
```

Let me check with @montelaidev if we can use the v3, but yes, that should be ok IMO, good catch :)

Reply:
Also, this PR is required for the on-going release, so we cannot align the `@metamask/accounts-controller` now, we might do it later.

Would this be ok for you?